test/wh_test_crypto.c

```diff
@@ -706,7 +949,16 @@ int whTest_CryptoClientConfig(whClientConfig* config)
         goto exit;
     }
 #endif /* HAVE_ECC */
+#endif
```

Comment:
dead code. Imagine you just want to remove the old code and instead call the new function

Reply:
yup. removed.